sys/include/net/gnrc/ipv6/util.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2018 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup net_gnrc_ipv6_util     GNRC IPv6 utility functions
+ * @ingroup  net_gnrc_ipv6
+ * @brief   GNRC IPv6 utility functions
+ * @{
+ *
+ * @file
+ * @brief   GNRC IPv6 utility functions
+ *
+ * @author  Martine Lenders <m.lenders@fu-berlin.de>
+ */
+#ifndef NET_GNRC_IPV6_UTIL_H
+#define NET_GNRC_IPV6_UTIL_H
+
+#include "net/ipv6/addr.h"
+#include "net/gnrc/netif.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Configures a prefix delegation lease that is provided by the server.
+ *
```

Comment:
Please add some details on what this does entail. From a user perspective: there is already `gnrc_ipv6_nib_pl_add()` so, why is this better / more feature-some?